pkg/spiffs/doc.txt

```diff
@@ -1,7 +1,6 @@
 /**
  * @defgroup pkg_spiffs   SPI flash file system
  * @ingroup  pkg
- * @ingroup  sys_fs
```

Comment:
There is already a spiffs group in `sys_fs` regarding RIOT integration. It looks inconsistent IMO as it appears twice in the  FS group in system.